docs/data/data-grid/overview/overview.md

```diff
@@ -43,6 +50,18 @@ import { DataGridPro } from '@mui/x-data-grid-pro';
 
 {{"demo": "DataGridProDemo.js", "defaultCodeOpen": false, "disableAd": true, "bg": "inline"}}
 
+#### Premium Plan [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-premium/)
+
+The Premium version contains all features from Pro, as well as exclusive features for data-rich applications.
+The features already available in this version are Row grouping and Excel export, with more coming.
+In the following demo, you can experiment with them.
+
+```js
+import { DataGridPremium } from '@mui/x-data-grid-premium';
+```
+
+{{"demo": "DataGridPremiumDemo.js", "defaultCodeOpen": false, "disableAd": true, "bg": "inline"}}
```

Comment:
Excel export fails in this demo:
<img width="659" alt="Screenshot 2022-05-05 at 12 20 03" src="https://user-images.githubusercontent.com/13808724/166904722-5631095d-d7b5-4bd6-b9b5-3e37752056e3.png">

Reply:
I'm looking into it

Reply:
https://github.com/mui/mui-x/pull/4774